panels/UserPanel.php

```diff
@@ -116,4 +124,19 @@ protected function dataToString($data)
 
         return VarDumper::export($data);
     }
+
+    /**
+     * Returns the array/object that should be set on [[\yii\widgets\DetailView::model]]
+     *
+     * @param IdentityInterface $identity
```

Comment:
Ah OK, so probably not worth the explicit type declaration then. Just changed it back to `mixed`.